raspi-config

```diff
@@ -3043,7 +3043,7 @@ do_advanced_menu() {
       "A4 Network Interface Names" "Enable/disable predictable network i/f names" \
       "A5 Network Proxy Settings" "Configure network proxy settings" \
       "A6 Boot Order" "Choose network or USB device boot" \
-      "A7 Bootloader Version" "Select latest or default boot ROM software" \
+      "A7 Bootloader Version" "Select bootloader EEPROM software" \
```

Comment:
Any reason not to make this "Choose latest or default bootloader EEPROM software" ?

Reply:
> Any reason not to make this "Choose latest or default bootloader EEPROM software" ?

Because `bootloader EEPROM` is longer than `boot ROM`, that menu description would be too big for the screen on the default 80x24 terminal if I left `latest or default` in there, so I removed it. (Whiptail does not wrap menu items, so you just end up losing characters off the end of the line). I think it is OK without this, although Ideally I would have preferred to retain it.